src/components/HeroSection.tsx

```diff
@@ -0,0 +1,46 @@
+          fill="#ffffff"
+        />
+      </svg>
+      <H2 className="text-white">無聊想找伴或參加活動嗎</H2>
```

Comment:
1. 內容少了一個問號
2. 待調整：手機版是 `H4`，PC 版是 `H2`
```suggestion
      <H2 className="text-white">無聊想找伴或參加活動嗎？</H2>
```

Reply:
fix on this commit: 75fb3d68c9678ba99d362222c9e74bb5b333bd61